airbyte-integrations/bases/base-normalization/dbt-project-template/macros/cross_db_utils/datatypes.sql

```diff
@@ -12,6 +12,10 @@
     varchar
 {%- endmacro -%}
 
+{%- macro oracle__type_json() -%}
+    varchar2(3000)
```

Comment:
Find max value to varchar2(MAX)

Reply:
Try to use `VARCHAR2 (23767)
   CONSTRAINT ensure_json CHECK (po_document IS JSON))`

From Oracle docs
```
CREATE TABLE j_purchaseorder
  (id          VARCHAR2 (32) NOT NULL PRIMARY KEY,
   date_loaded TIMESTAMP (6) WITH TIME ZONE,
   po_document VARCHAR2 (23767)
   CONSTRAINT ensure_json CHECK (po_document IS JSON));
```